### PR TITLE
Add ability to play live video

### PR DESCRIPTION
**Pull request by BohdanBilonoh**
Was added button for watching live video
screen shot 2017-09-21 at 11 33 57 am

### CODE REVIEW

File: Examples/tvOS Integration/Podfile.lock [hunk 1]
```diff
@@ -1,15 +1,15 @@
 PODS:
-  - OneMobileSDK-Swift3 (2.6.1375):
-    - VideoRenderer (= 1.1)
-  - VideoRenderer (1.1)
+  - OneMobileSDK-Swift3 (2.16.1780):
```
Comment (AndriiMoskvin) [inline Examples/tvOS Integration/Podfile.lock]: Beta have to be here.
I don't see part that reflects that `isFinished` is fixed.

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: Examples/tvOS Integration/Podfile.lock (hunk 1 above)
Comment (BohdanBilonoh) [reply Examples/tvOS Integration/Podfile.lock]: @AndriiMoskvin 👍

File: Examples/tvOS Integration/tvOS Integration/ViewController.swift [hunk 2]
```diff
@@ -12,9 +12,21 @@ class ViewController: UIViewController {
         super.viewDidLoad()
     }
     
+    var isContentFinished: Bool?
```
Comment (AndriiMoskvin) [inline Examples/tvOS Integration/tvOS Integration/ViewController.swift]: Why this is on level of class?

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: Examples/tvOS Integration/tvOS Integration/ViewController.swift (hunk 2 above)
Comment (BohdanBilonoh) [reply Examples/tvOS Integration/tvOS Integration/ViewController.swift]: Moved to func scope

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**